compilers/concrete-compiler/compiler/lib/Support/CompilerEngine.cpp

```diff
@@ -472,6 +472,16 @@ CompilerEngine::compile(llvm::SourceMgr &sm, Target target, OptionalLib lib) {
   if (target == Target::SIMULATED_TFHE)
     return std::move(res);
 
+  // Stats
+  if (res.feedback) {
+    // TFHE Stats
+    if (mlir::concretelang::pipeline::extractTFHEStatistics(
+            mlirContext, module, this->enablePass, res.feedback.value())
+            .failed()) {
+      return errorDiag("Extracting TFHE statistics failed");
+    }
+  }
```

Comment:
this should at least move before simulation

Reply:
Also before batch

Reply:
Done.